userpanel/modules/info/style/bclean/templates/userinfobox.html

```diff
@@ -17,24 +17,26 @@
 		{/if}
 		{if $rights.info.edit_addr || $rights.info.edit_addr_ack || $rights.info.edit_contact || $rights.info.edit_contact_ack}
 			{if $rights.info.edit_addr || $rights.info.edit_addr_ack || $rights.info.edit_contact || $rights.info.edit_contact_ack}
-				<button id="updatedatabtn" type="button" class="btn btn-primary btn-sm" data-toggle="modal" data-target="#updateuserinfobox" accesskey="A">
-					{trans("Actualize data")}
-				</button>
+			<button id="updatedatabtn" type="button" class="btn btn-primary btn-sm" data-toggle="modal" data-target="#updateuserinfobox" accesskey="A">
+				{trans("Actualize data")}
+			</button>
 			{/if}
 		{/if}
 	</div>
 	<ul class="list-group list-group-flush">
 		<li class="list-group-item">
-			<strong>{trans("ID")}:</strong>
-			{$userinfo.id|string_format:"%04d"}
+			<strong>{trans("ID")}:</strong>{$userinfo.id|string_format:"%04d"}
```

Comment:
Czy tu nie chodziło Ci o wymuszenie braku łamania wiersza tekstu?

Reply:
Tak i usunięcie zbędnej spacji.

Reply:
To może wystarczyłaby klasa **nobr**?

Reply:
Niestety wrzucenie nobr nie usunie zbędnej spacji.